components/webpack/webpack.config.js

```diff
@@ -119,7 +119,25 @@ module.exports = async function (env, argv) {
         {
           test: /\.(ttf|eot|ico|svg|png|jpg|jpeg|gif)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
           loader: 'file-loader'
-        }]
+        },
+        {
+          test: (input) => {
```

Comment:
It doesn't seem clear to me what this rule is for - is it to get around something that works in webpack 5 but not 4? If so, could we leave a comment explaining something like that so we know what to do when we upgrade to webpack 5?

Reply:
Good point. I will add a comment.

Reply:
@petemill added here: https://github.com/brave/brave-core/pull/13943/commits/acc9d2a1e17932a5261c189735883d92d30413c3